app/points/calculate-weighting.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function (weightingPercentage) {
+  var weighting
+  if (weightingPercentage >= 100) {
+    weighting = 0
+  } else if (weightingPercentage <= 0) {
+    weighting = 1
+  } else {
+    weighting = (100 - weightingPercentage) / 100
+  }
+  return weighting
+}
```

Comment:
i prefer single exit from habit/debugging when it gets more than a few lines. Lets pick one and update anything we come across 👍